rmf_task/test/mock/MockDelivery.cpp

```diff
@@ -49,7 +50,9 @@ auto MockDelivery::make_activator() -> Activator
 
 auto MockDelivery::Active::backup() const -> Backup
 {
-  return Backup::make(_backup_seq++, "Hello, I am a backup");
+  std::stringstream ss;
+  ss << this->active_phase()->tag()->id() << std::endl;
```

Comment:
This change is fine, but it breaks [this test](https://github.com/open-rmf/rmf_task/blob/e1c2d51f7f32daf36e5298247656ed1e84b667bd/rmf_task/test/unit/test_Activator.cpp#L69). The test expectation should be updated to reflect this implementation change.